fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFab.kt

```diff
@@ -67,7 +77,11 @@ fun TabsTrayFab(
         }
     }
 
-    Box(Modifier.fillMaxSize()) {
+    Box(
+        Modifier
+            .fillMaxSize()
+            .semantics { invisibleToUser() },
```

Comment:
We need this to allow the a11y services access to the tabs the same way we allow in the XML tabs tray.